NineOlds dependency switched to support lib #62

redwerk opened the pull request.

pedrovgs commented on gradle.properties:
```diff
@@ -13,7 +13,7 @@ POM_LICENCE_DIST=repo
-ANDROID_TARGET_SDK_VERSION=20
+ANDROID_BUILD_TOOLS_VERSION=23.0.2
+ANDROID_COMPILE_SDK_VERSION=23
+ANDROID_TARGET_SDK_VERSION=19
```
Why did you change this target sdk to api 19?

pedrovgs commented on sample/build.gradle:
```diff
@@ -10,7 +10,7 @@ dependencies {
   compile 'com.github.pedrovgs:renderers:1.0.9'
   compile 'com.google.android.gms:play-services:4.1.32'
-  compile 'com.actionbarsherlock:actionbarsherlock:4.4.0@aar'
+  compile "com.android.support:appcompat-v7:23.1.1"
```
Can you use `'` instead of `"`?

pedrovgs commented:
Hi @redwerk can you also review the usage of the method `getSupportActionBar()` The code is not compiling because the compiler can't find this method.

redwerk commented:
Hi @pedrovgs ,
Sorry for the long delay response.
I've changed quotes as you wanted.
Changing from target sdk from 19 to 20 was a mistake, sorry about that. Fixed too.
getSupportActionBar() now can be found be compiler.

Thanks!

pedrovgs commented:
Sorry for the delay @redwerk, I've been out for some time. You have some checkstyle issues. Can you review it and fix it to get the build working on travis?

redwerk commented:
@pedrovgs, I've updated this fix, but it fails again and again, i guess you have to update gradle wrapper to 2.8 version, because everything is ok on my machine and device.

pedrovgs commented:
Seems related to the Travis-CI configuration. This is the failure output `> failed to find target with hash string 'android-23' in: /usr/local/android-sdk` Can you take a look?
